dashboard/cron.py

```diff
@@ -263,18 +267,16 @@ def update_assignment(self):
         # delete all records in assignment table
         status += deleteAllRecordInTable("assignment")
 
-        # return string with concatenated SQL insert result
-        returnString = ""
-
         # loop through multiple course ids
         for UDW_course_id in settings.DEFAULT_UDW_COURSE_IDS:
-            assignment_sql="with assignment_info as " \
-                           "(select ad.due_at AS due_date,ad.due_at at time zone 'utc' at time zone 'America/New_York' as local_date," \
-                           "ad.title AS name,af.course_id AS course_id,af.assignment_id AS id," \
-                           "af.points_possible AS points_possible,af.assignment_group_id AS assignment_group_id" \
-                           " from assignment_fact af inner join assignment_dim ad on af.assignment_id = ad.id where af.course_id='" + UDW_course_id + "'" \
-                                                                                                                                                      "and ad.visibility = 'everyone' and ad.workflow_state='published')" \
-                                                                                                                                                      "select * from assignment_info"
+            assignment_sql = f"""with assignment_info as
```

Comment:
What is the syntax of f""" stands for? Formatted string value?

Reply:
Yeah, they're fstrings, new in Python 3.6. I kind of think it's pretty clean to have them like this.

https://realpython.com/python-f-strings/

We could change it to .format, but I think either .format or fstrings are easier to read than inline concatenation.

Reply:
Nice to know!